packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2777,6 +2788,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private verifyCanSubmitOps() {
+        if (this.mc.config.getBoolean("Fluid.ContainerRuntime.DisableOpReentryCheck") === true) {
+            return;
+        }
+
+        if (this.runWithoutOpsCalls > 0) {
+            throw new UsageError("Op was submitted from within a `runWithoutOps` callback");
+        }
```

Comment:
We will probably break existing scenarios, won't we ? I wonder if we should simply log an event to see how often it happens.

Reply:
This is opt-in now, so the point is moot.